db/queries/reporting/queries.py

```diff
@@ -35,36 +35,43 @@ def get_general_status_applications_report(
 KEY_REPORT_MAPPING = [
     {
         "form_name": "organisation-information",
+        "form_name_cy": "gwybodaeth-am-y-sefydliad",
         "key": "WWWWxy",
         "return_field": "eoi_reference",
     },
     {
         "form_name": "organisation-information",
+        "form_name_cy": "gwybodaeth-am-y-sefydliad",
         "key": "YdtlQZ",
         "return_field": "organisation_name",
     },
     {
         "form_name": "organisation-information",
+        "form_name_cy": "gwybodaeth-am-y-sefydliad",
         "key": "lajFtB",
         "return_field": "organisation_type",
     },
     {
         "form_name": "asset-information",
+        "form_name_cy": "gwybodaeth-am-yr-ased",
         "key": "yaQoxU",
         "return_field": "asset_type",
     },
     {
         "form_name": "project-information",
+        "form_name_cy": "gwybodaeth-am-y-prosiect",
         "key": "yEmHpp",
         "return_field": "geography",
```

Comment:
@Adam-W1 Why is the return_field "geography"? I would have assumed it returned the project name field?

Reply:
unsure tbh @rayhan15243  I didnt come up with this naming convention - it does return the UK address but maybe the rrequriement was that this is called "geography" in the report

Reply:
@Adam-W1 Ah okay cool. I've approved